android/src/main/java/com/RNFetchBlob/RNFetchBlobFS.java

```diff
@@ -665,10 +665,14 @@ static void exists(String path, Callback callback) {
             }
         }
         else {
-            path = normalizePath(path);
-            boolean exist = new File(path).exists();
-            boolean isDir = new File(path).isDirectory();
-            callback.invoke(exist, isDir);
+            try {
+                path = normalizePath(path);
+                boolean exist = new File(path).exists();
+                boolean isDir = new File(path).isDirectory();
+                callback.invoke(exist, isDir);
+            } catch (NullPointerException e) {
```

Comment:
There are multiple items in the `try` block that could trigger this exception. Probably unwise to hide them all since it could make finding future bugs more challenging. Perhaps better to check if the path was valid before using it further, as opposed to catching exceptions later.